integrations/drift/drift.go

```diff
@@ -90,7 +90,6 @@ func (api driftAPIProduction) request(method string, endpoint string, payload []
 	req.Header.Set("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	defer resp.Body.Close()
```

Comment:
When I add the retry-code, the app just crashes if I don't move this as accessing `resp.Body`will cause an invalid memory address crash (say if there is no internet access). I didn't figure out any other way to avoid the crash.